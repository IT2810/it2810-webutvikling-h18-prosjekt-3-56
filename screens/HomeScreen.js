import React from 'react';
import {Text, View, StyleSheet} from 'react-native';
import Pedometer from '../features/MotivatingPedometer.js';
import Joke from '../components/Joke.js';

export default class HomeScreen extends React.Component {

  static navigationOptions = {  // Display options for this screen. Used by the navigator that renders it
    title: 'Home'
  };

  render() {
    return (
      <View style={styles.container}>
        <View style = {{flex:1}}>
          <Joke/>
        </View>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff'
  }
});
