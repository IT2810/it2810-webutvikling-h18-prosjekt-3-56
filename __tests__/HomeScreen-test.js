import React from 'react';
import HomeScreen from '../screens/HomeScreen.js';
import renderer from 'react-test-renderer';
import ShallowRenderer from 'react-test-renderer/shallow';

it("renders correctly", () => {
	const tree = renderer.create(<HomeScreen />).toJSON();
	expect(tree).toMatchSnapshot();
})