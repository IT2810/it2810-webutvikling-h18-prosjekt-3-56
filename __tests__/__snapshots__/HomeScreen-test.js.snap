// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "justifyContent": "center",
        "marginTop": "3%",
      }
    }
  >
    <View
      style={
        Object {
          "height": "100%",
          "marginHorizontal": "3%",
          "width": "94%",
        }
      }
    >
      <View
        onLayout={[Function]}
        style={
          Object {
            "height": "100%",
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "column-reverse",
          "height": "90%",
          "position": "absolute",
          "width": "100%",
          "zIndex": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "height": "40%",
            "justifyContent": "center",
            "marginBottom": "3%",
            "width": "100%",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "fontSize": undefined,
              "fontWeight": "400",
            }
          }
        >
             
          100 /
        </Text>
        <TextInput
          allowFontScaling={true}
          onSubmitEditing={[Function]}
          placeholder="1000"
          style={
            Object {
              "fontSize": undefined,
              "fontWeight": "400",
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
            "height": "45%",
            "justifyContent": "center",
            "width": "100%",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        />
      </View>
    </View>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "borderBottomColor": "black",
        "borderBottomWidth": 0.5,
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "center",
        "opacity": 1,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "backgroundColor": "white",
            "margin": 3,
            "shadowColor": "#000",
            "shadowOffset": Object {
              "height": 1,
              "width": 2,
            },
            "shadowOpacity": 0.8,
            "shadowRadius": 4,
            "textAlign": "center",
          },
          Object {
            "fontSize": 35,
            "textAlign": "center",
          },
          Object {},
        ]
      }
    >
      
    </Text>
  </View>
</View>
`;
